src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -260,14 +266,21 @@ class FunSpec private constructor(builder: Builder) {
       typeVariables += typeVariable
     }
 
-    fun receiver(receiverType: TypeName) = apply {
+    @JvmOverloads fun receiver(receiverType: TypeName, kdoc: CodeBlock? = null) = apply {
```

Comment:
Since the default field value is `CodeBlock.EMPTY`, would it make sense to change the `kdoc` type to non-nullable and use `CodeBlock.EMPTY` as default? That would prevent people from explicitly passing `null` and we won't need to null-check in the method body.

Reply:
Good point! The same thing makes sense for `returns` also. I can address that in a follow up.